admin/conf/application.conf

```diff
@@ -42,3 +42,5 @@ application.langs="en"
 # ~~~~~
 # You can disable evolutions if needed
 # evolutionplugin=disabled
+com.lucidchart.piezo.heartbeatFile="/tmp/piezo/workerHeartbeatFile"
+healthCheck.worker.minutesBetween=2
```

Comment:
I don't know that we need this that strict. Why don't we make it 5 minutes to allow for some server load that prevents the main thread from running for a short period?